stories/required_with_context/ComponentWithRef.js

```diff
@@ -0,0 +1,36 @@
+import React, { PropTypes } from 'react'
```

Comment:
Yeah the storybooks/storyshots repo is running at `"react": "^15.4.1"`. I didn't want to update dependencies and stuff here as the repo is deprecated ...